packages/wallets/src/features/cashier/modules/WithdrawalLocked/WithdrawalLockedContent.tsx

```diff
@@ -25,65 +25,45 @@ export const getWithdrawalLimitReachedDesc = ({
     poiNeedsVerification,
     poiStatus,
 }: TWithdrawalLimitReachedDescProps) => {
-    if (poiNeedsVerification && poiStatus === 'none')
-        return {
-            description: (
-                <WalletText align='center'>
-                    <Trans
-                        components={[<WalletLink href='/account/proof-of-identity' key={0} variant='bold' />]}
-                        defaults='You have reached the withdrawal limit. Please upload your <0>proof of identity</0> to lift the limit to continue your withdrawal.'
-                    />
-                </WalletText>
-            ),
-        };
+    let description = null;
+    const generateDescription = (description: string, components?: JSX.Element[]) => (
+        <WalletText align='center'>
+            <Trans components={components} defaults={`You have reached the withdrawal limit. ${description}`} />
+        </WalletText>
+    );
```

Comment:
@lubega-deriv 
since we have the same code on line 77, 
could we move
```
const generateDescription = (description: string, components?: JSX.Element[]) => (
        <WalletText align='center'>
            <Trans components={components} defaults={`You have reached the withdrawal limit. ${description}`} />
        </WalletText>
    );
```
this outside the components and reuse it ? 🤔

Reply:
Updated